doc/sphinxext/prepare_gallery.py

```diff
@@ -228,11 +262,16 @@ def prepare_gallery(app=None):
         # Create readme file
         if example.readme.startswith('file:'):
             filename = example.readme.split('file:')[1].strip()
-            shutil.copy(Path(examples_dir, filename), Path(folder,
-                                                           'README.rst'))
+            # shutil.copy(Path(examples_dir, filename), Path(folder,
+            #                                                'README.rst'))
```

Comment:
Remove instead of commenting?

Reply:
done done in 343100f12f614161f024dc85f1c76c83a9c09683